src/tests/ftest/network/cart_self_test.py

```diff
@@ -78,6 +79,14 @@ def setUp(self):
             self.server_managers[0].get_config_value("provider")
         self.cart_env["OFI_INTERFACE"] = \
             self.server_managers[0].get_config_value("fabric_iface")
+
+        self.cart_env["DEFAULT_DAOS_AGENT_DRPC_SOCK"] = \
+            "/var/run/daos_agent/daos_agent.sock"
```

Comment:
i dont think this one is envariable

Reply:
3c0a08a9e8a501e02612f38e9f4fb85456daf917